spec/features/admin/admin_sees_order_details_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+feature 'Logged in admin' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.